axelor-quality/src/main/resources/views/TrackingNumberCharacteristic.xml

```diff
@@ -34,7 +35,8 @@
           onChange="action-tracking-number-method-characterisctic-assign-conforms"/>
       </panel>
       <panel name="commentPanel" colSpan="12">
-        <field name="trackingNumberComment" colSpan="12"/>
+        <field name="trackingNumberComment" colSpan="12"
+          onChange="action-quality-method-tracking-number-characterisctic-assign-conforms"/>
```

Comment:
Please check this action, I think you meant 
`action-tracking-number-method-characterisctic-assign-conforms` also did not see but there is a typo in characteris**C**tic
And the action name should be `action-tracking-number-characteristic-method-assign-conforms`